doc/install.rst

```diff
@@ -4,26 +4,28 @@ Installation
 
 Check dependencies
 ------------------
-You only need ``NumPy``, ``SciPy`` for the package and additionally ``scikit-learn``
-if you would like to run the examples. We recommend
-``Python >= 2.6``,
-``scikit-learn >= 0.17``,
-``NumPy >= 1.6.1`` and ``SciPy >= 0.14``.
+We currently support ``Python 3.5+``.
+
+For the package: ``numpy>=1.11``, ``scipy>=0.17``, ``scikit-learn>=0.18``
+
+Additionally, for examples: ``pandas>=0.20``
 
 Both `Canopy <https://www.enthought.com/products/canopy/>`__
 and `Anaconda <https://www.continuum.io/downloads>`__
 ship with a recent version of all these packages.
 
+Additionally, for development, tests and coverage: ``pytest``, ``pytest-cov``, ``coverage``, ``flake8``
+
+Additionally, for building documentation: ``sphinx``, ``sphinx-gallery``, ``pillow``, ``numpydoc``, ``matplotlib``
```

Comment:
same as above. Move to developer documentation?

Reply:
I wanted one place for all dependencies to be present; so it's ok to duplicate across parts of the documentation